src/pages/CrewRecruit/CrewRecruit.js

```diff
@@ -1,21 +1,89 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import Banner from "./components/Banner";
 import HowToCrew from "./components/HowToCrew";
 import Interview from "./components/Interview";
 import Intro from "./components/Intro";
 import Recruit from "./components/Recruit/Recruit";
 import Tab from "./components/Tab";
 import Work from "./components/Work";
+
 const CrewRecruit = () => {
+  const [currentSection, setCurrentSection] = useState("");
+
+  const handleScroll = () => {
+    const scrollY = window.scrollY;
+    const intro = document.getElementById("intro");
+    const introRect = intro.getBoundingClientRect();
+    const introPosition = Math.floor(scrollY + (introRect.y - 75));
+
+    const work = document.getElementById("work");
+    const workRect = work.getBoundingClientRect();
+    const workPosition = Math.floor(scrollY + (workRect.y - 75));
+
+    const interview = document.getElementById("interview");
+    const interviewRect = interview.getBoundingClientRect();
+    const interviewPosition = Math.floor(scrollY + (interviewRect.y - 75));
+
+    const howToCrew = document.getElementById("howToCrew");
+    const howToCrewRect = howToCrew.getBoundingClientRect();
+    const howToCrewPosition = Math.floor(scrollY + (howToCrewRect.y - 75));
+
+    const recruit = document.getElementById("recruit");
+    const recruitRect = recruit.getBoundingClientRect();
+    const recruitPosition = Math.floor(scrollY + recruitRect.y);
```

Comment:
```suggestion
    const introTop = intro.current?.offsetTop - 75;
    const workTop = work.current?.offsetTop - 75;
    const interviewTop = interview.current?.offsetTop - 75;
    const howToCrewTop = howToCrew.current?.offsetTop - 75;
    const recruitTop = recruit.current?.offsetTop - 75;
```

DOM에 직접 접근 하는 방식 보다 useRef 를 사용해서 구현하는 방식이
조금 더 좋은 방법으로 생각 됩니다

useRef를 적용해서 위의 코드를 사용할 수 있도록 구현해주세요!